vue/src/App.vue

```diff
@@ -65,19 +65,14 @@
         </div>
         <!-- Charts Grid -->
         <div class="charts-container">
-          <div class="section-header">
-            <h2>{{ getCurrentOperation.name }}</h2>
-            <p class="section-description">{{ getCurrentOperation.description }}</p>
-          </div>
-          
           <div class="charts-grid">
             <router-view v-slot="{ Component }">
-              <transition name="fade" mode="out-in">
+              <keep-alive>
```

Comment:
Should we set a max number of components to cache here?  The components can be holding onto a _lot_ of data.

Reply:
Absolutely, setting a max limit is a good idea. What limit would you suggest for the maximum number of components to cache?

Reply:
Dunno - 10?

Reply:
Sounds good